src/modules/snowflake/data-connector/template.yaml

```diff
@@ -0,0 +1,100 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. 2021
+# SPDX-License-Identifier: Apache-2.0
+
+AWSTemplateFormatVersion: '2010-09-09'
+
+# This template is using AWS SAM template
+# Syntax Ref: https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-specification.html.
+Transform: AWS::Serverless-2016-10-31
+
+Globals:
+  Function:
+    Timeout: 60
+    MemorySize: 2048
+    Runtime: python3.8
+
+Resources:
```

Comment:
wondering if would be helpful to add descriptions so customers know what each of these resources are fow